tests/end_to_end/end_to_end.cpp

```diff
@@ -506,6 +507,111 @@ droppacket_test(ebpf_execution_type_t execution_type)
     bpf_object__close(unique_object.release());
 }
 
+void
+irql_low_test(ebpf_execution_type_t execution_type)
+{
+    _test_helper_end_to_end test_helper;
+    test_helper.initialize();
+
+    int result;
+    const char* error_message = nullptr;
+    bpf_object_ptr unique_object;
+    fd_t program_fd;
+    bpf_link_ptr link;
+
+    single_instance_hook_t hook(EBPF_PROGRAM_TYPE_XDP, EBPF_ATTACH_TYPE_XDP);
```

Comment:
Since the XDP type will move to the xdp-for-windows repo, is it possible to test this with some program type that won't move?

Reply:
It's the only program information provider that has min/max == DISPATCH_LEVEL, so not really.

Reply:
Then we should add one to the sample driver.  Why doesn't BPF_PROG_TYPE_XDP_TEST have min/max == DISPATCH_LEVEL?